docs/Containers/Syncthing.md

```diff
@@ -21,7 +21,7 @@ The web UI can be found on  `yourip:8384`
 
 Configuration data is available under `/config` containers directroy and mapped to `./volumes/syncthing/config` . 
 
-The data you want to sync must be under the same directory as configuration one. By default, Syncthing creates a `Sync` subdirectory
+The data you want to sync must be under `/app` directory.
```

Comment:
The app directory is inside the container, end users will see `./volumes/syncthing/data`. The default share is named Sync. Other sync folders will also appear under data.

Reply:
> > * Network mode network_mode: host NOT set: Unsolved but runs ok. (i would like to be settable by user in menus but i still need to investigate how to do it)
> 
> I don't see any reason not to use host networking. And official Syncthing docs state that not doing so will result in slow local transfer rates. IOTstack is aimed for beginners and should be (or at least aim for) an it-just-works experience.

As i understand, if you set network mode to host value, the container really control the host network allowing to modify it as it wants. You can not restrict the ports exposed. Thats why i would like to manage someway in options menu. 

I have tested at home and I get about 8MB/s under a fiber with 100Mb/s. I think it is good enough for 90% users. 

If none agree with me i can change to host mode but right now i think it is not really a very good practice at all.